cpp/tests/io/parquet_test.cpp

```diff
@@ -6534,7 +6537,7 @@ TEST_F(ParquetReaderTest, FilterFloatNAN)
   auto col0 = cudf::test::fixed_width_column_wrapper<float>(elements, elements + num_rows);
   auto col1 = cudf::test::fixed_width_column_wrapper<double>(elements, elements + num_rows);
 
-  cudf::test::print(col0);
+  if constexpr (print_enabled) { cudf::test::print(col0); }
```

Comment:
Is this line even necessary? Seems this was added for debugging something specific and was just forgotten.
It does not seem difficult to re-add the line if debugging is needed in the future.
Also, future debugging would probably want to print a different variable in a different place and enabling the constant could produce confusing output.

Reply:
```suggestion
```